src/app/window-handler.ts

```diff
@@ -828,23 +835,22 @@ export class WindowHandler {
             });
         }
 
+        logger.info('window-handler: createScreenSharingIndicatorWindow, displayId: ' + displayId);
         if (displayId !== '') {
-            const displays = electron.screen.getAllDisplays();
-
-            displays.forEach((element) => {
-                if (displayId === element.id.toString()) {
-                    logger.info(`window-handler: element:`, element);
-                    if (isWindowsOS) {
-                        this.execCmd(this.screenShareIndicatorFrameUtil, [ displayId ]);
-                    } else if (isLinux) {
-                        this.createScreenSharingFrameWindow('screen-sharing-frame',
-                        element.workArea.width,
-                        element.workArea.height,
-                        element.workArea.x,
-                        element.workArea.y);
+            if (isLinux) {
+                const displays = electron.screen.getAllDisplays();
+                displays.forEach((element) => {
+                    logger.info('window-handler: element.id.toString(): ' + element.id.toString());
```

Comment:
please consider using String interpolation for log statements with variable values.